test/ETO/ETOCommitment.js

```diff
@@ -1182,13 +1219,17 @@ contract("ETOCommitment", ([, admin, company, nominee, ...investors]) => {
   });
 
   describe("special ETO configurations", () => {
-    it("reverts overflow on internal equity token 56 bit", async () => {
-      // totals of equity tokens are stored in 56 bits number
-      const two = new web3.BigNumber(2);
-      const one = new web3.BigNumber(1);
+    it.skip("reverts overflow on internal equity token 128 bit", async () => {
+      // totals of equity tokens are stored in 128 bits number
+      // overflowing that is practically impossible, we'll need 2^32 tickets of 2^96 tickets (which is max)
```

Comment:
> .... 2^32 tickets of 2^96 tickets ...

 should be of 2^96 tokens

Reply:
ok comment fixed

Reply:
`// overflowing that is practically impossible, we'll need 2^32 investments with 2^96 ticket amount (which is max)`